src/core/MIGRATION_EXAMPLES.md

```diff
@@ -466,3 +466,52 @@ elsewhere.
 In most cases, the most convenient way to access these APIs will be via the
 [AppMountContext](/docs/development/core/public/kibana-plugin-public.appmountcontext.md)
 object passed to your application when your app is mounted on the page.
+
+## Render HTML Content
+
+You can return a blank HTML page bootstrapped with the core application bundle from an HTTP route handler
+via the `rendering` context.
+
+```ts
+router.get(
+  { path: '/', validate: false },
+  (context, request, response) => {
+    const { http, rendering } = context.core;
+
+    return response.ok({
+      body: await rendering.render(),
+      headers: {
+        'content-security-policy': http.csp.header,
+      },
+    });
+  }
+);
+```
+
+You can also specify to exclude user data from the bundle metadata:
```

Comment:
Can you add clarification of what user data is and why you would want to exclude it?